scripts/bootloader/provision.py

```diff
@@ -41,17 +40,17 @@ def add_hw_counters(provision_data, num_counter_slots_version, mcuboot_counters_
     assert num_counter_slots_version % 2 == 0, "--num-counters-slots-version must be an even number"
     assert mcuboot_counters_slots    % 2 == 0, "--mcuboot-counters-slots     must be an even number"
 
-    provision_data += struct.pack('H', 1) # Type "counter collection"
-    provision_data += struct.pack('H', num_counters) # Could be 0, 1, or 2
+    provision_data += struct.pack('I', 1) # Type "counter collection"
```

Comment:
need get back to `short int`. this line and all bellow

Reply:
Fixed